std_msgs/Header header
uint32 seqOriginal
sensor_msgs/Image im
int64 cameraFrameId
int8 howManyBoxes
yolorosort/singleBBox[] boxesWithAll
float32[] sortedBoxes
